app/plugins/Wallet/js/lifecycle.js

```diff
@@ -46,8 +48,20 @@ function updateAddrTxn(addr) {
 	}, 'update-history');
 }
 
-// Update wallet summary in header
+// Update transaction history and addresses
 addResultListener('update-status', function(wallet) {
+	refreshRate = finalRefreshRate; // slow down after first successful call
+
+	// Show correct lock status.
+	// TODO: If the wallet is encrypted, prompt with a pw.
```

Comment:
@Mingling94 I think we commented at the same time.

After you unlock the wallet the first time, it's viable to see your balance and txn history while the wallet is locked. I'm planning some upgrades so that this is also true even if you haven't yet unlocked the wallet. Also, it will eventually be okay to mine + do some forms of hosting and renting while the wallet is locked. None of these features are yet available, but they are in the pipeline.